events.html

```diff
@@ -0,0 +1,212 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>About Us | BAMP</title>
+        <link rel="stylesheet" href="style/index.css">
+        <script src="scripts/index.js" type="module" defer></script>
+    </head>
+
+    <body>
+        <div class="header">
+            <a class="header--linkLogo" href="/index.html">
+                <img class="header--logo" src="images/logo-mini-color.svg">
+            </a>
+            <a class="header--link" href="/artwork.html">Artwork</a>
+            <a class="header--link" href="/about_us.html">About Us</a>
+            <a class="header--link" href="/programs.html">Programs</a>
+            <a class="header--link" href="/events.html">Events</a>
+            <a class="header--link" href="/services.html">Services</a>
+            <a class="header--link" href="#">Shop</a>
+            <a class="header--link" href="/get_involved.html">Get Involved</a>
+            <a class="header--link header--link-tight" href="#"><span class="btnLink btnLink-small">Get Updates</span></a>
+            <a class="header--link header--link-tight" href="#"><span class="btnLink btnLink-red btnLink-small">Donate</span></a>
+            <button id="hamburger" class="header--hamburger">☰</button>
+        </div>
+    </body>
+
+    <div id="navModal" class="navModal">
+        <button id="closeNavModal" class="navModal--close">×</button>
+        <ul class="navModal--linkList">
+            <li><a class="navModal--link" href="/index.html">Home</a></li>
+            <li><a class="navModal--link" href="/artwork.html">Artwork</a></li>
+            <li><a class="navModal--link" href="/about_us.html">About Us</a></li>
+            <li><a class="navModal--link" href="/programs.html">Programs</a></li>
+            <li><a class="navModal--link" href="/events.html">Events</a></li>
+            <li><a class="navModal--link" href="/services.html">Services</a></li>
+            <li><a class="navModal--link" href="#">Shop</a></li>
+            <li><a class="navModal--link" href="/get_involved.html">Get Involved</a></li>
+            <li><a class="btnLink btnLink-large">Get Updates</a></li>
+            <li><a class="btnLink btnLink-red btnLink-large">Donate</a></li>
+        </ul>
+    </div>
+
+    <p class="event--title l-center">Events</p>
```

Comment:
I think you should change this to `<h2 class="sectionHeading sectionHeading-spaced .sectionHeading-centered">`

Reply:
agreed, but no dot before the last one